cypress/e2e/rents.spec.cy.ts

```diff
@@ -0,0 +1,91 @@
+describe("template spec", () => {
+  it("should visit localhost:3000, go to create new rent, create a new rent, go to details, modify it and delete it.", () => {
+    cy.visit("http://localhost:3000/");
```

Comment:
You could set this url as a dynamic var, to make it easier to handle by establishing it in your `cypress.config.ts` check how to do it [here](https://docs.cypress.io/api/cypress-api/config)